### PR TITLE
finalize admin login screen and implement functions

### PR DESCRIPTION
**Pull request by dev7ch**

### CODE REVIEW

File: modules/admin/src/resources/dist/css/Test-Env › Login_files/login.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.card {
```
Comment (nadar) [inline modules/admin/src/resources/dist/css/Test-Env › Login_files/login.css]: What is the idea behind this file?

File: modules/admin/src/resources/dist/css/Test-Env › Login_files/login.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+var observeLogin = function(form, url, secureUrl) {
```
Comment (nadar) [inline modules/admin/src/resources/dist/css/Test-Env › Login_files/login.js]: Same here?

File: modules/admin/src/resources/dist/css/Test-Env › Login_files/saved_resource.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (nadar) [inline modules/admin/src/resources/dist/css/Test-Env › Login_files/saved_resource.html]: Not sure what is the purpose of this file?

**Comment by TheMaaarc** [comment]
Silvan will open a second pull request with only the required files.

**Pull request closed without merging**